src/app/core/entity-components/entity-details/form/form.component.html

```diff
@@ -112,7 +112,10 @@
           </mat-form-field>
         </div>
 
-        <div *ngIf="row.input === 'select'">
+        <div *ngIf="row.input === 'select'"
+          [matTooltip]="row.tooltip"
+          [matTooltipPosition]="row.tooltipPosition"
```

Comment:
couldn't you add this to the parent div (row 9) and automatically make it available for all types?

Reply:
Yes, that would work; thank you for that input.